src/permission/authorization/PermissionMap.js

```diff
@@ -256,15 +256,11 @@ function PermPermissionMap($q, $log, $injector, PermTransitionProperties, PermRo
       } else if (angular.isFunction(redirection)) {
         redirectionMap[permission] = redirection;
         redirectionMap[permission].$inject = ['rejectedPermission', 'transitionProperties'];
-      }
-
-      if (angular.isObject(redirection)) {
+      } else if (angular.isObject(redirection)) {
```

Comment:
I had to do this here since isObject([]) === true. It was re-defining the property as this function instead of the array-syntax injected function.